### PR TITLE
Added youtube resource for android development

### LINKED ISSUE #1148: [Add] Add youtube resource for android development
### Site URL

https://www.youtube.com/playlist?list=PLjVLYmrlmjGdDps6HAwOOVoAtBPAgIOXL

### Description of the above provided site

This android tutorials in Hindi explains how android applications are built using Android studio. This android course will teach you everything you need to learn to create your first Android Application!

### Category

Youtube

### Sub Category

Youtube

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by madhurafulkar**
## Fixes Issue

Closes #1148 

## Changes proposed

Added youtube resource for android development

## Screenshots

Screenshot 2023-07-08 100011

### CODE REVIEW

File: database/youtube/android.json [hunk 1]
```diff
@@ -46,5 +46,13 @@
     "category": "youtube",
     "subcategory": "android",
     "language": "hindi"
+  },
+  {
+    "name": "Android Development Course| Neat Roots",
+    "description": "In this Android Java Course you can learn complete Android Development by Full Stack Android Developer",
```
Comment (CBID2) [inline database/youtube/android.json:52]: [Proofreading]
```suggestion
    "description": "In this Android Java Course, you learn Android Development to become a Full Stack Android Developer",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just made a proofreading suggestion. Other than that, good start! :)

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Pull request merged**